Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4EheNQ7D4c27QswDGPiDpGtf9rfQF1HauEmyd+/REsZ/AAAAoh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyeEgZx/eT83ILSktQiICOvpCg/Jye1SDdLHwCRPP8lzAAAAA==[0mcontroller-j in /root/.jenkins/workspace/cyp-test-nic-simple
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/cyp-test-nic-simple/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/cyp-test-nic-simple/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (set args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/conf/change_args.py '{"bmc_conf":{"user":"debuguser","password":"0penBmc1"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-21.02/","sut_conf":{"bmc_ip":"10.239.183.128","board_type":"commercial","host":"10.239.182.209","pf1":"ens785f2","os":"RHEL","PF1_NIC_type":"4*25"},"pkg_generator":{"host":"10.239.182.79","pf1":"ens787f2"},"drive_info":{"drive_version":"1.6.4"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics -s -m nicSimple --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_simple_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 19 items / 11 deselected / 8 selected

npxtester/Tests/testNics/test_mtu.py 2022-04-12 00:25:49 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_Jumbo_Packet_CVL_00_25_49/test_PI_Networking_Jumbo_Packet_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_Jumbo_Packet_CVL_00_25_49
2022-04-12 00:25:49 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-12 00:25:49 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-12 00:25:49 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-12 00:25:49 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-12 00:25:49 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:25:49 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:25:49 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 00:25:49 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:25:49 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:25:50 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 00:25:50 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 00:25:50 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 00:25:53 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 00:25:53 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 00:25:58 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 00:25:58 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 00:26:01 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 00:26:10 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:26:15 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 00:26:15 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 00:26:18 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 00:26:23 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:26:28 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 00:26:31 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:26:31 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:26:39 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 00:26:39 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 00:26:47 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 mtu 1500
2022-04-12 00:26:47 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:26:55 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 mtu 1500
2022-04-12 00:26:55 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:27:03 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 00:27:03 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 18  bytes 2384 (2.3 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 00:27:12 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 14:55:03 2022 from 10.112.97.60
[root@npx-cypsilver1-rel84 ~]# scapy
[39mINFO: Can't import matplotlib. Won't be able to plot.[0m
[39mINFO: Can't import PyX. Won't be able to use psdump() or pdfdump().[0m
[33m[1mWARNING: IPython not available. Using standard Python shell instead.
AutoCompletion, History are disabled.[0m[0m
[32m[1m                                      [0m[34m[1m[0m
[32m[1m                     aSPY//YASa       [0m[34m[1m[0m
[32m[1m             apyyyyCY//////////YCa    [0m[34m[1m   |[0m
[32m[1m            sY//////YSpcs  scpCY//Pp  [0m[34m[1m   | Welcome to Scapy[0m
[32m[1m ayp ayyyyyyySCP//Pp           syY//C [0m[34m[1m   | Version 2.4.5[0m
[32m[1m AYAsAYYYYYYYY///Ps              cY//S[0m[34m[1m   |[0m
[32m[1m         pCCCCY//p          cSSps y//Y[0m[34m[1m   | https://github.com/secdev/scapy[0m
[32m[1m         SPPPP///a          pP///AC//Y[0m[34m[1m   |[0m
[32m[1m              A//A            cyP////C[0m[34m[1m   | Have fun![0m
[32m[1m              p///Ac            sC///a[0m[34m[1m   |[0m
[32m[1m              P////YCpc           A//A[0m[34m[1m   | Craft packets like it is your last[0m
[32m[1m       scccccp///pSP///p          p//Y[0m[34m[1m   | day on earth.[0m
[32m[1m      sY/////////y  caa           S//P[0m[34m[1m   |                      -- Lao-Tze[0m
[32m[1m       cayCyayP//Ya              pY/Ya[0m[34m[1m   |[0m
[32m[1m        sY/PsY////YCc          aC//Yp [0m
[32m[1m         sc  sccaCY//PCypaapyCP//YSs  [0m
[32m[1m                  spCPY//////YPSps    [0m
[32m[1m                       ccaacs         [0m
[32m[1m                                      [0m
[34m[1m>>> [0m
2022-04-12 00:27:19 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="40:a6:b7:72:cc:ca")/IP(len=46)/Raw(load='X'*14480)],iface="ens787f2")
.
Sent 1 packets.
[34m[1m>>> [0m
2022-04-12 00:27:22 - test_mtu.py - [line:92] - root - INFO - success to send package to network
2022-04-12 00:27:26 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="40:a6:b7:72:cc:ca")/IP(len=46)/Raw(load='X'*15500)],iface="ens787f2")
Traceback (most recent call last):
  File "/usr/lib64/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 463, in sendp
    **kargs
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 397, in _send
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 360, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy/arch/linux.py", line 569, in send
    return SuperSocket.send(self, x)
  File "/usr/local/lib/python3.6/site-packages/scapy/supersocket.py", line 108, in send
    return self.outs.send(sx)
OSError: [Errno 90] Message too long
[34m[1m>>> [0m
2022-04-12 00:27:29 - test_mtu.py - [line:102] - root - INFO - result is right
.2022-04-12 00:27:32 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 mtu 1500
2022-04-12 00:27:32 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:27:40 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 mtu 1500
2022-04-12 00:27:40 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:27:48 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 00:27:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:27:56 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-12 00:27:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:28:01 - test_mtu.py - [line:51] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-12 00:28:01 - test_mtu.py - [line:53] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-12/test_PI_Networking_Jumbo_Packet_CVL_00_25_49
2022-04-12 00:28:04 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-12 00:28:04 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:28:09 - test_mtu.py - [line:55] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-12 00:28:09 - test_mtu.py - [line:56] - root - INFO - this is Test_Nics teardown
2022-04-12 00:28:09 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-12 00:28:09 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_MTU_set_CVL_00_28_09/test_PI_Networking_MTU_set_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_MTU_set_CVL_00_28_09
2022-04-12 00:28:09 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_MTU_set_CVL case ============================
2022-04-12 00:28:09 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-12 00:28:09 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-12 00:28:09 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-12 00:28:09 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:28:09 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:28:09 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 00:28:09 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:28:09 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:28:10 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 00:28:10 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 00:28:10 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 00:28:13 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 00:28:13 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 00:28:18 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 00:28:18 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 00:28:21 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 00:28:31 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:28:36 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 00:28:36 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 00:28:39 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 00:28:43 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:28:48 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 00:28:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:28:51 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:28:59 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 00:29:00 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 00:29:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 mtu 1500
2022-04-12 00:29:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:29:16 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 mtu 2000
2022-04-12 00:29:16 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:29:24 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 00:29:24 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2204 (2.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 00:29:33 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Tue Apr 12 00:39:04 2022 from 10.112.97.60
[root@npx-cypsilver1-rel84 ~]# scapy
[39mINFO: Can't import matplotlib. Won't be able to plot.[0m
[39mINFO: Can't import PyX. Won't be able to use psdump() or pdfdump().[0m
[33m[1mWARNING: IPython not available. Using standard Python shell instead.
AutoCompletion, History are disabled.[0m[0m
[32m[1m                                      [0m[34m[1m[0m
[32m[1m                     aSPY//YASa       [0m[34m[1m[0m
[32m[1m             apyyyyCY//////////YCa    [0m[34m[1m   |[0m
[32m[1m            sY//////YSpcs  scpCY//Pp  [0m[34m[1m   | Welcome to Scapy[0m
[32m[1m ayp ayyyyyyySCP//Pp           syY//C [0m[34m[1m   | Version 2.4.5[0m
[32m[1m AYAsAYYYYYYYY///Ps              cY//S[0m[34m[1m   |[0m
[32m[1m         pCCCCY//p          cSSps y//Y[0m[34m[1m   | https://github.com/secdev/scapy[0m
[32m[1m         SPPPP///a          pP///AC//Y[0m[34m[1m   |[0m
[32m[1m              A//A            cyP////C[0m[34m[1m   | Have fun![0m
[32m[1m              p///Ac            sC///a[0m[34m[1m   |[0m
[32m[1m              P////YCpc           A//A[0m[34m[1m   | Craft packets before they craft[0m
[32m[1m       scccccp///pSP///p          p//Y[0m[34m[1m   | you.[0m
[32m[1m      sY/////////y  caa           S//P[0m[34m[1m   |                      -- Socrate[0m
[32m[1m       cayCyayP//Ya              pY/Ya[0m[34m[1m   |[0m
[32m[1m        sY/PsY////YCc          aC//Yp [0m
[32m[1m         sc  sccaCY//PCypaapyCP//YSs  [0m
[32m[1m                  spCPY//////YPSps    [0m
[32m[1m                       ccaacs         [0m
[32m[1m                                      [0m
[34m[1m>>> [0m
2022-04-12 00:29:40 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="40:a6:b7:72:cc:ca")/IP(len=46)/Raw(load='X'*18890)],iface="ens787f2")
.
Sent 1 packets.
[34m[1m>>> [0m
2022-04-12 00:29:43 - test_mtu.py - [line:132] - root - INFO - success to send package to network
2022-04-12 00:29:47 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="40:a6:b7:72:cc:ca")/IP(len=46)/Raw(load='X'*20010)],iface="ens787f2")
Traceback (most recent call last):
  File "/usr/lib64/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 463, in sendp
    **kargs
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 397, in _send
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 360, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy/arch/linux.py", line 569, in send
    return SuperSocket.send(self, x)
  File "/usr/local/lib/python3.6/site-packages/scapy/supersocket.py", line 108, in send
    return self.outs.send(sx)
OSError: [Errno 90] Message too long
[34m[1m>>> [0m
2022-04-12 00:29:50 - test_mtu.py - [line:142] - root - INFO - result is right
.2022-04-12 00:29:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 mtu 1500
2022-04-12 00:29:53 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:30:01 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 mtu 1500
2022-04-12 00:30:02 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:30:10 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 00:30:10 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:30:18 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-12 00:30:18 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:30:23 - test_mtu.py - [line:51] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-12 00:30:23 - test_mtu.py - [line:53] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-12/test_PI_Networking_MTU_set_CVL_00_28_09
2022-04-12 00:30:26 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-12 00:30:26 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:30:31 - test_mtu.py - [line:55] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-12 00:30:31 - test_mtu.py - [line:56] - root - INFO - this is Test_Nics teardown
2022-04-12 00:30:31 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_MTU_set_CVL case ============================
                                  [ 25%]
npxtester/Tests/testNics/test_nics.py 2022-04-12 00:30:31 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-12 00:30:31 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_silver_Networking_ConnectivityEnableDisable_00_30_31/test_silver_Networking_ConnectivityEnableDisable.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_silver_Networking_ConnectivityEnableDisable_00_30_31
2022-04-12 00:30:31 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_silver_Networking_ConnectivityEnableDisable case ============================
2022-04-12 00:30:31 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-12 00:30:31 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-12 00:30:31 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:30:31 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:30:31 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 00:30:31 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:30:31 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:30:31 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 00:30:31 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 00:30:34 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:30:34 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:30:42 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-12 00:30:42 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-12 00:30:47 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-12 00:30:47 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 00:30:50 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 00:30:50 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 00:30:55 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 00:30:55 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 00:30:58 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 00:31:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:31:13 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 00:31:13 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 00:31:16 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 00:31:21 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:31:26 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 00:31:29 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:31:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:31:37 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 00:31:37 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 00:31:45 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-12 00:31:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:31:50 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-12 00:31:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 00:31:53 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 00:32:01 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-12 00:32:01 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6461414959  bytes 9782567061838 (8.8 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 249370508  bytes 18516680140 (17.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 00:32:09 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:32:09 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:32:17 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-12 00:32:17 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:32:26 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Tue Apr 12 00:43:28 2022 from 10.112.97.60
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.045 ms
2022-04-12 00:32:27 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.016 ms
2022-04-12 00:32:28 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.023 ms
2022-04-12 00:32:29 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.018 ms
2022-04-12 00:32:30 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.020 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 120ms
rtt min/avg/max/mdev = 0.016/0.024/0.045/0.011 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:32:39 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 down
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:32:48 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 up
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:32:54 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:32:54 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:33:03 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.041 ms
2022-04-12 00:33:04 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.014 ms
2022-04-12 00:33:05 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.019 ms
2022-04-12 00:33:06 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.020 ms
2022-04-12 00:33:07 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.028 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 97ms
rtt min/avg/max/mdev = 0.014/0.024/0.041/0.010 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:33:16 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 down
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:33:25 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 up
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:33:31 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:33:31 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:33:40 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.034 ms
2022-04-12 00:33:41 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.026 ms
2022-04-12 00:33:42 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.029 ms
2022-04-12 00:33:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.022 ms
2022-04-12 00:33:44 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.015 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 138ms
rtt min/avg/max/mdev = 0.015/0.025/0.034/0.007 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:33:53 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 down
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:34:02 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 up
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:34:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:34:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:34:17 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.042 ms
2022-04-12 00:34:18 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.036 ms
2022-04-12 00:34:19 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.016 ms
2022-04-12 00:34:20 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.024 ms
2022-04-12 00:34:21 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.019 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 115ms
rtt min/avg/max/mdev = 0.016/0.027/0.042/0.011 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:34:30 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 down
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:34:39 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 up
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:34:45 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:34:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:34:54 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.041 ms
2022-04-12 00:34:55 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.023 ms
2022-04-12 00:34:56 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.022 ms
2022-04-12 00:34:57 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.019 ms
2022-04-12 00:34:58 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.029 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 92ms
rtt min/avg/max/mdev = 0.019/0.026/0.041/0.010 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:35:07 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 down
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:35:16 - sshInstance.py - [line:99] - root - INFO - ifconfig ens785f2 up
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:35:22 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:35:22 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:35:31 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
64 bytes from 192.168.10.10: icmp_seq=1 ttl=64 time=0.046 ms
2022-04-12 00:35:32 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=2 ttl=64 time=0.025 ms
2022-04-12 00:35:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=3 ttl=64 time=0.025 ms
2022-04-12 00:35:34 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=4 ttl=64 time=0.025 ms
2022-04-12 00:35:35 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.10: icmp_seq=5 ttl=64 time=0.027 ms

--- 192.168.10.10 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 132ms
rtt min/avg/max/mdev = 0.025/0.029/0.046/0.010 ms
[root@npx-cyp7-rh82 ~]# 
.2022-04-12 00:35:41 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 00:35:41 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:35:49 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 00:35:50 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:35:58 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-12 00:35:58 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:36:06 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-12 00:36:06 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:36:14 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-12 00:36:14 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:36:19 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-12 00:36:19 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-12/test_silver_Networking_ConnectivityEnableDisable_00_30_31
2022-04-12 00:36:22 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-12 00:36:22 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:36:27 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-12 00:36:27 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 00:36:27 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-12 00:36:27 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_silver_Networking_ConnectivityEnableDisable case ============================
2022-04-12 00:36:27 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_silver_ConnectivityReboot_00_36_27/test_silver_ConnectivityReboot.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_silver_ConnectivityReboot_00_36_27
2022-04-12 00:36:27 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_silver_ConnectivityReboot case ============================
2022-04-12 00:36:27 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-12 00:36:27 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-12 00:36:27 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:36:27 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:36:27 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 00:36:27 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:36:27 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:36:27 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 00:36:27 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 00:36:30 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:36:30 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:36:38 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-12 00:36:39 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-12 00:36:44 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-12 00:36:44 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 00:36:47 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 00:36:47 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 00:36:52 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 00:36:52 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 00:36:55 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 00:37:03 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:37:08 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 00:37:08 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 00:37:11 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 00:37:15 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:37:20 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 00:37:23 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:37:23 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:37:31 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 00:37:31 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 00:37:39 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-12 00:37:39 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:37:44 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-12 00:37:47 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 00:37:47 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 00:37:55 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-12 00:37:55 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6461415154  bytes 9782567088951 (8.8 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 249370534  bytes 18516685831 (17.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 00:38:03 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-12 00:38:03 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:38:11 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:38:11 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:38:16 - test_nics.py - [line:138] - root - INFO - -- reboot 1 times --
2022-04-12 00:38:20 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Tue Apr 12 00:49:24 2022 from 10.112.97.60
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.464 ms
2022-04-12 00:38:21 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.170 ms
2022-04-12 00:38:22 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.188 ms
2022-04-12 00:38:23 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.196 ms
2022-04-12 00:38:24 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.209 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 77ms
rtt min/avg/max/mdev = 0.170/0.245/0.464/0.110 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:38:31 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.546 ms
2022-04-12 00:38:32 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.212 ms
2022-04-12 00:38:33 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.192 ms
2022-04-12 00:38:34 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.219 ms
2022-04-12 00:38:35 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.210 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 135ms
rtt min/avg/max/mdev = 0.192/0.275/0.546/0.137 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:38:44 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-12 00:38:47 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-12 00:39:47 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:39:52 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:39:52 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:39:52 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:39:52 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 114ms
2022-04-12 00:39:52 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:39:52 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:39:52 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:39:57 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:40:02 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:40:02 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:02 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:40:02 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 88ms
2022-04-12 00:40:02 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:02 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:02 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:40:07 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:40:12 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:40:12 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:12 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:40:12 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 125ms
2022-04-12 00:40:12 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:12 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:12 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:40:17 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:40:23 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:40:23 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:23 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:40:23 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 99ms
2022-04-12 00:40:23 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:23 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:23 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:40:28 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:40:33 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:40:33 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:33 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:40:33 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 138ms
2022-04-12 00:40:33 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:33 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:33 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:40:38 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:40:43 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:40:43 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.395 ms
2022-04-12 00:40:43 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:43 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:40:43 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 1 received, 80% packet loss, time 112ms
2022-04-12 00:40:43 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.395/0.395/0.395/0.000 ms
2022-04-12 00:40:43 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:43 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:40:48 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.361 ms
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.328 ms
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.355 ms
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.381 ms
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.391 ms
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 85ms
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.328/0.363/0.391/0.025 ms
2022-04-12 00:40:53 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:40:53 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:40:53 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 00:43:53 - test_nics.py - [line:138] - root - INFO - -- reboot 2 times --
2022-04-12 00:43:53 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:43:53 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:43:53 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 00:43:56 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:43:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:44:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:44:04 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:44:13 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Tue Apr 12 00:53:46 2022
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.441 ms
2022-04-12 00:44:14 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.202 ms
2022-04-12 00:44:15 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.182 ms
2022-04-12 00:44:16 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.193 ms
2022-04-12 00:44:17 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.194 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 80ms
rtt min/avg/max/mdev = 0.182/0.242/0.441/0.100 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:44:24 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.543 ms
2022-04-12 00:44:25 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.191 ms
2022-04-12 00:44:26 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.198 ms
2022-04-12 00:44:27 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.188 ms
2022-04-12 00:44:28 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.193 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 137ms
rtt min/avg/max/mdev = 0.188/0.262/0.543/0.141 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:44:37 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-12 00:44:40 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-12 00:45:40 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:45:45 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:45:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:45:45 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:45:45 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 82ms
2022-04-12 00:45:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:45:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:45:45 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:45:50 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:45:55 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:45:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:45:55 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:45:55 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 120ms
2022-04-12 00:45:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:45:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:45:55 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:46:00 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:46:05 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:46:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:05 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:46:05 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 94ms
2022-04-12 00:46:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:05 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:46:10 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:46:15 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:46:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:15 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:46:15 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 133ms
2022-04-12 00:46:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:15 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:46:20 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:46:25 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:46:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:25 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:46:25 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 107ms
2022-04-12 00:46:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:25 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:46:30 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:46:35 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:46:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:35 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:46:35 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 82ms
2022-04-12 00:46:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:35 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:46:40 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.356 ms
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.331 ms
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.373 ms
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.370 ms
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.371 ms
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 120ms
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.331/0.360/0.373/0.019 ms
2022-04-12 00:46:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:46:45 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:46:45 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 00:49:45 - test_nics.py - [line:138] - root - INFO - -- reboot 3 times --
2022-04-12 00:49:45 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:49:45 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:49:45 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 00:49:48 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:49:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:49:56 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:49:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:50:05 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Tue Apr 12 00:59:45 2022
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.470 ms
2022-04-12 00:50:06 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.220 ms
2022-04-12 00:50:07 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.208 ms
2022-04-12 00:50:08 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.206 ms
2022-04-12 00:50:09 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.231 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 92ms
rtt min/avg/max/mdev = 0.206/0.267/0.470/0.101 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:50:16 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.550 ms
2022-04-12 00:50:17 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.268 ms
2022-04-12 00:50:18 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.196 ms
2022-04-12 00:50:19 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.208 ms
2022-04-12 00:50:20 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.193 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 86ms
rtt min/avg/max/mdev = 0.193/0.283/0.550/0.136 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:50:29 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-12 00:50:32 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-12 00:51:32 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:51:37 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:51:37 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:37 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:51:37 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 117ms
2022-04-12 00:51:37 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:37 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:37 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:51:42 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:51:47 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:51:47 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:47 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:51:47 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 90ms
2022-04-12 00:51:47 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:47 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:47 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:51:52 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:51:57 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:51:57 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:57 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:51:57 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 129ms
2022-04-12 00:51:57 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:57 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:51:57 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:52:02 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:52:07 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:52:07 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:07 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:52:07 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 103ms
2022-04-12 00:52:07 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:07 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:07 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:52:12 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:52:17 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:52:17 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:17 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:52:17 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 78ms
2022-04-12 00:52:17 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:17 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:17 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:52:22 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:52:27 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:52:27 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:27 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:52:27 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 116ms
2022-04-12 00:52:27 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:27 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:27 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:52:32 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.387 ms
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.398 ms
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.373 ms
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.368 ms
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.383 ms
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 91ms
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.368/0.381/0.398/0.026 ms
2022-04-12 00:52:37 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:52:37 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:52:37 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 00:55:38 - test_nics.py - [line:138] - root - INFO - -- reboot 4 times --
2022-04-12 00:55:38 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 00:55:38 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 00:55:38 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 00:55:41 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 00:55:41 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:55:49 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 00:55:49 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 00:55:58 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Tue Apr 12 01:05:36 2022
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.447 ms
2022-04-12 00:55:59 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.202 ms
2022-04-12 00:56:00 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.193 ms
2022-04-12 00:56:01 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.206 ms
2022-04-12 00:56:02 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.208 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 85ms
rtt min/avg/max/mdev = 0.193/0.251/0.447/0.098 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:56:09 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.508 ms
2022-04-12 00:56:10 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.197 ms
2022-04-12 00:56:11 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.204 ms
2022-04-12 00:56:12 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.189 ms
2022-04-12 00:56:13 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.192 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 79ms
rtt min/avg/max/mdev = 0.189/0.258/0.508/0.125 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 00:56:22 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-12 00:56:25 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-12 00:57:25 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:57:30 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:57:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:30 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:57:30 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 86ms
2022-04-12 00:57:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:30 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:57:35 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:57:40 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:57:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:40 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:57:40 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 125ms
2022-04-12 00:57:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:40 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:57:45 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:57:50 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:57:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:50 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:57:50 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 99ms
2022-04-12 00:57:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:57:50 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:57:55 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:58:00 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:58:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:00 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:58:00 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 138ms
2022-04-12 00:58:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:00 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:58:05 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:58:10 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:58:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:10 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:58:10 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 112ms
2022-04-12 00:58:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:10 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:58:15 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:58:20 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:58:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:20 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:58:20 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 86ms
2022-04-12 00:58:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:20 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:58:25 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.378 ms
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.378 ms
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.334 ms
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.336 ms
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.377 ms
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 125ms
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.334/0.360/0.378/0.029 ms
2022-04-12 00:58:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 00:58:30 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 00:58:30 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 01:01:30 - test_nics.py - [line:138] - root - INFO - -- reboot 5 times --
2022-04-12 01:01:30 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:01:30 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:01:30 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:01:33 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:01:33 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:01:41 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 01:01:41 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:01:50 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Tue Apr 12 01:11:30 2022
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.473 ms
2022-04-12 01:01:51 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.198 ms
2022-04-12 01:01:52 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.196 ms
2022-04-12 01:01:53 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.198 ms
2022-04-12 01:01:54 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.198 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 82ms
rtt min/avg/max/mdev = 0.196/0.252/0.473/0.111 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 01:02:01 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.509 ms
2022-04-12 01:02:02 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.209 ms
2022-04-12 01:02:03 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.198 ms
2022-04-12 01:02:04 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.202 ms
2022-04-12 01:02:05 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.190 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 140ms
rtt min/avg/max/mdev = 0.190/0.261/0.509/0.125 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-12 01:02:14 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-12 01:02:17 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-12 01:03:17 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:03:22 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:03:22 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:22 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:03:22 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 118ms
2022-04-12 01:03:22 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:22 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:22 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:03:27 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:03:32 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:03:32 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:32 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:03:32 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 92ms
2022-04-12 01:03:32 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:32 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:32 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:03:37 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:03:42 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:03:42 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:42 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:03:42 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 131ms
2022-04-12 01:03:42 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:42 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:42 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:03:47 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:03:52 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:03:52 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:52 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:03:52 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 105ms
2022-04-12 01:03:52 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:52 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:03:52 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:03:57 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:04:02 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:04:02 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:04:02 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:04:02 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 79ms
2022-04-12 01:04:02 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:04:02 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:04:02 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:04:07 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:04:12 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:04:12 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:04:12 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:04:12 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 117ms
2022-04-12 01:04:12 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:04:12 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:04:12 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:04:17 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.379 ms
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.404 ms
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.356 ms
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.382 ms
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.396 ms
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 91ms
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.356/0.383/0.404/0.024 ms
2022-04-12 01:04:22 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:04:22 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:04:22 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
.2022-04-12 01:07:26 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:07:26 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:07:26 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:07:26 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:07:26 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 01:07:29 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:07:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:07:37 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-12 01:07:37 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:07:45 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-12 01:07:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:07:53 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-12 01:07:53 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:07:58 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-12 01:07:58 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-12/test_silver_ConnectivityReboot_00_36_27
2022-04-12 01:08:01 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-12 01:08:01 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:08:06 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-12 01:08:06 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 01:08:06 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-12 01:08:06 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_silver_ConnectivityReboot case ============================
2022-04-12 01:08:06 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_gold_PI_Networking_SRIOV_CheckDevice_01_08_06/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_gold_PI_Networking_SRIOV_CheckDevice_01_08_06
2022-04-12 01:08:06 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-12 01:08:06 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-12 01:08:06 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-12 01:08:06 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:08:06 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:08:06 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:08:06 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:08:06 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:08:06 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 01:08:06 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 01:08:09 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:08:09 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:08:18 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-12 01:08:18 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-12 01:08:23 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-12 01:08:23 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 01:08:26 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 01:08:26 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 01:08:31 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 01:08:31 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 01:08:34 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 01:08:46 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:08:51 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 01:08:51 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 01:08:54 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 01:08:58 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:09:03 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 01:09:06 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:09:06 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:09:14 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 01:09:14 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 01:09:22 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-12 01:09:22 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:09:27 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-12 01:09:30 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 01:09:31 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:09:39 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-12 01:09:39 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6461415522  bytes 9782567142382 (8.8 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 249370683  bytes 18516710671 (17.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:09:47 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:09:47 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:00 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11

2022-04-12 01:10:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:13 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 down
2022-04-12 01:10:13 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:21 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 up
2022-04-12 01:10:21 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:29 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 192.168.10.12
2022-04-12 01:10:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:37 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.12
2022-04-12 01:10:41 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.458 ms
64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.196 ms
64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.189 ms
64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.191 ms
64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.223 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4081ms
rtt min/avg/max/mdev = 0.189/0.251/0.458/0.104 ms

2022-04-12 01:10:57 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 down
2022-04-12 01:10:58 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:11:06 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 up
2022-04-12 01:11:06 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:11:14 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 192.168.10.13
2022-04-12 01:11:14 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:11:22 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.13
2022-04-12 01:11:28 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
From 192.168.10.11 icmp_seq=1 Destination Host Unreachable
From 192.168.10.11 icmp_seq=2 Destination Host Unreachable
From 192.168.10.11 icmp_seq=3 Destination Host Unreachable
From 192.168.10.11 icmp_seq=4 Destination Host Unreachable
From 192.168.10.11 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 4116ms
pipe 4

F2022-04-12 01:11:39 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:11:42 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:11:50 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:11:50 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:11:58 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-12 01:11:58 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:12:06 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-12 01:12:06 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:12:14 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-12 01:12:14 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:12:19 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-12 01:12:19 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-12/test_gold_PI_Networking_SRIOV_CheckDevice_01_08_06
2022-04-12 01:12:22 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-12 01:12:22 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:12:27 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-12 01:12:27 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 01:12:27 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-12 01:12:27 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-12 01:12:27 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_InternetProtocolv4v6_01_12_27/test_PI_Networking_InternetProtocolv4v6.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_InternetProtocolv4v6_01_12_27
2022-04-12 01:12:27 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_InternetProtocolv4v6 case ============================
2022-04-12 01:12:27 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-12 01:12:27 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-12 01:12:27 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:12:28 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:12:28 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:12:28 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:12:28 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:12:28 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 01:12:28 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 01:12:31 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:12:31 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:12:39 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-12 01:12:39 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-12 01:12:44 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-12 01:12:44 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 01:12:47 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 01:12:47 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 01:12:52 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 01:12:52 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 01:12:55 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 01:13:05 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:13:10 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 01:13:10 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 01:13:13 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 01:13:17 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:13:22 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 01:13:25 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:13:25 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:13:33 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 01:13:33 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 01:13:41 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-12 01:13:41 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:13:46 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-12 01:13:49 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 01:13:49 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 16  bytes 2134 (2.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:13:57 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-12 01:13:57 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6461415769  bytes 9782567181768 (8.8 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 249370721  bytes 18516717675 (17.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:14:05 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 01:14:05 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:14:13 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-12 01:14:13 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:14:21 - sshInstance.py - [line:50] - root - INFO - ping -w 5 192.168.10.11
2022-04-12 01:14:26 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.453 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.195 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.221 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.205 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.185 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 114ms
rtt min/avg/max/mdev = 0.185/0.251/0.453/0.103 ms

2022-04-12 01:14:34 - sshInstance.py - [line:50] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
2022-04-12 01:14:39 - sshInstance.py - [line:54] - root - INFO - ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.509 ms
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.188 ms
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.189 ms
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.182 ms
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.185 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 125ms
rtt min/avg/max/mdev = 0.182/0.250/0.509/0.130 ms

.2022-04-12 01:14:47 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:14:47 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:14:55 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:14:55 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:15:03 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-12 01:15:03 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:15:11 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-12 01:15:11 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:15:19 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-12 01:15:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:15:24 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-12 01:15:24 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-12/test_PI_Networking_InternetProtocolv4v6_01_12_27
2022-04-12 01:15:27 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-12 01:15:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:15:32 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-12 01:15:32 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 01:15:32 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-12 01:15:32 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_InternetProtocolv4v6 case ============================
2022-04-12 01:15:32 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_Link_State_Check_01_15_32/test_PI_Networking_Link_State_Check.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_Link_State_Check_01_15_32
2022-04-12 01:15:32 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Link_State_Check case ============================
2022-04-12 01:15:32 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-12 01:15:32 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-12 01:15:32 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:15:32 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:15:32 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:15:32 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:15:32 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:15:33 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 01:15:33 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 01:15:36 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:15:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:15:44 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-12 01:15:44 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-12 01:15:49 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-12 01:15:49 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 01:15:52 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 01:15:52 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 01:15:57 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 01:15:57 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 01:16:00 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 01:16:10 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:16:15 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 01:16:15 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 01:16:18 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 01:16:22 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:16:27 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 01:16:30 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:16:31 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:16:39 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 01:16:39 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 01:16:47 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-12 01:16:47 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:16:52 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-12 01:16:55 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 01:16:55 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 16  bytes 2134 (2.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:17:03 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-12 01:17:03 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6461415815  bytes 9782567187724 (8.8 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 249370758  bytes 18516724612 (17.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:17:11 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-04-12 01:17:12 - sshInstance.py - [line:54] - root - INFO - 0000:4b:00.2

2022-04-12 01:17:20 - sshInstance.py - [line:50] - root - INFO - lspci -vvs 00:4b:00.2 | grep LnkCap:
2022-04-12 01:17:20 - sshInstance.py - [line:54] - root - INFO - 		LnkCap:	Port #0, Speed 16GT/s, Width x16, ASPM not supported

2022-04-12 01:17:28 - sshInstance.py - [line:50] - root - INFO - lspci -vvs 00:4b:00.2 | grep LnkSta:
2022-04-12 01:17:28 - sshInstance.py - [line:54] - root - INFO - 		LnkSta:	Speed 16GT/s, Width x16, TrErr- Train- SlotClk+ DLActive- BWMgmt- ABWMgmt-

2022-04-12 01:17:33 - test_nics.py - [line:283] - root - INFO - --check the speed and width of the ethernet ens785f2
2022-04-12 01:17:33 - test_nics.py - [line:287] - root - INFO - the linkCap and LnkSta value are matched. The speed and width value are: 16GT/s and Width x16.
.2022-04-12 01:17:36 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:17:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:17:44 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:17:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:17:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-12 01:17:53 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:18:01 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-12 01:18:01 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:18:09 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-12 01:18:09 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:18:14 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-12 01:18:14 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-12/test_PI_Networking_Link_State_Check_01_15_32
2022-04-12 01:18:17 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-12 01:18:17 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:18:22 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-12 01:18:22 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 01:18:22 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-12 01:18:22 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_Link_State_Check case ============================
2022-04-12 01:18:22 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_ConnectivityShutdown_01_18_22/test_PI_Networking_ConnectivityShutdown.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_PI_Networking_ConnectivityShutdown_01_18_22
2022-04-12 01:18:22 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_ConnectivityShutdown case ============================
2022-04-12 01:18:22 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-12 01:18:22 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-12 01:18:22 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:18:22 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:18:22 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:18:22 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:18:22 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:18:22 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 01:18:22 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 01:18:25 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:18:26 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:18:34 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-12 01:18:34 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-12 01:18:39 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-12 01:18:39 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 01:18:42 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 01:18:42 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 01:18:47 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 01:18:47 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 01:18:50 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 01:19:02 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:19:07 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 01:19:07 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 01:19:10 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 01:19:15 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:19:20 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 01:19:23 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:19:23 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:19:31 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 01:19:31 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 01:19:39 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-12 01:19:39 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:19:44 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-12 01:19:47 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 01:19:47 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 16  bytes 2134 (2.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:19:55 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-12 01:19:55 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6461415836  bytes 9782567190428 (8.8 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 249370760  bytes 18516724866 (17.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:20:03 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-12 01:20:04 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:20:12 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 01:20:12 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:20:20 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11

2022-04-12 01:20:24 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.451 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.231 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.200 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.194 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.205 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 82ms
rtt min/avg/max/mdev = 0.194/0.256/0.451/0.098 ms

2022-04-12 01:20:29 - power.py - [line:18] - root - INFO - checking the machine status
2022-04-12 01:20:29 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status
....
2022-04-12 01:20:29 - commonFunc.py - [line:30] - root - INFO - Chassis Power is on
2022-04-12 01:20:29 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:20:29 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status

2022-04-12 01:20:29 - power.py - [line:22] - root - INFO - Chassis Power is on

2022-04-12 01:20:29 - test_nics.py - [line:328] - root - INFO - Chassis Power is on

2022-04-12 01:20:29 - sshInstance.py - [line:143] - root - INFO - ping sut, please wait ......
2022-04-12 01:21:29 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.399 ms
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.373 ms
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.287 ms
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.326 ms
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.378 ms
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 129ms
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.287/0.352/0.399/0.045 ms
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:21:34 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:21:34 - power.py - [line:39] - root - INFO - starting to power off the machine....
2022-04-12 01:21:34 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power off
....
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - Chassis Power Control: Down/Off
2022-04-12 01:21:34 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:21:34 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power off

2022-04-12 01:21:34 - power.py - [line:44] - root - INFO - Chassis Power Control: Down/Off

2022-04-12 01:21:34 - test_nics.py - [line:331] - root - INFO - Chassis Power Control: Down/Off

2022-04-12 01:21:44 - power.py - [line:18] - root - INFO - checking the machine status
2022-04-12 01:21:44 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status
....
2022-04-12 01:21:44 - commonFunc.py - [line:30] - root - INFO - Chassis Power is off
2022-04-12 01:21:44 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:21:44 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status

2022-04-12 01:21:44 - power.py - [line:22] - root - INFO - Chassis Power is off

2022-04-12 01:21:44 - test_nics.py - [line:335] - root - INFO - the machine is powering off....
2022-04-12 01:21:44 - power.py - [line:27] - root - INFO - starting to power on the machine....
2022-04-12 01:21:44 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power on
....
2022-04-12 01:21:44 - commonFunc.py - [line:30] - root - INFO - Chassis Power Control: Up/On
2022-04-12 01:21:44 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:21:44 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power on

2022-04-12 01:21:44 - power.py - [line:32] - root - INFO - Chassis Power Control: Up/On

2022-04-12 01:21:44 - test_nics.py - [line:341] - root - INFO - Chassis Power Control: Up/On

2022-04-12 01:21:54 - sshInstance.py - [line:143] - root - INFO - ping sut, please wait ......
2022-04-12 01:22:55 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:23:00 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:23:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:00 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:23:00 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 85ms
2022-04-12 01:23:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:00 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:23:05 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:23:10 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:23:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:10 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:23:10 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 123ms
2022-04-12 01:23:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:10 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:23:15 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:23:20 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:23:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:20 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:23:20 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 97ms
2022-04-12 01:23:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:20 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:23:25 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:23:30 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:23:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:30 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:23:30 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 135ms
2022-04-12 01:23:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:30 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:23:35 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:23:40 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:23:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:40 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:23:40 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 109ms
2022-04-12 01:23:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:40 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:23:45 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:23:50 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:23:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:50 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:23:50 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 84ms
2022-04-12 01:23:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:23:50 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:23:55 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:24:00 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:24:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:00 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:24:00 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 123ms
2022-04-12 01:24:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:00 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:24:05 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:24:10 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:24:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:10 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:24:10 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 97ms
2022-04-12 01:24:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:10 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:24:15 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:24:20 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:24:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:20 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:24:20 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 135ms
2022-04-12 01:24:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:20 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:24:25 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:24:30 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:24:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:30 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:24:30 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 109ms
2022-04-12 01:24:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:30 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:24:35 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:24:40 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:24:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:40 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:24:40 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 82ms
2022-04-12 01:24:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:40 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:24:45 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:24:50 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:24:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:50 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:24:50 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 121ms
2022-04-12 01:24:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:24:50 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:24:55 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:25:00 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:25:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:25:00 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:25:00 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 96ms
2022-04-12 01:25:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:25:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:25:00 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:25:05 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.395 ms
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.409 ms
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.384 ms
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.379 ms
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.373 ms
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 134ms
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.373/0.388/0.409/0.012 ms
2022-04-12 01:25:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:25:10 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-12 01:30:10 - power.py - [line:18] - root - INFO - checking the machine status
2022-04-12 01:30:10 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status
....
2022-04-12 01:30:10 - commonFunc.py - [line:30] - root - INFO - Chassis Power is on
2022-04-12 01:30:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-12 01:30:10 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status

2022-04-12 01:30:10 - power.py - [line:22] - root - INFO - Chassis Power is on

2022-04-12 01:30:10 - test_nics.py - [line:347] - root - INFO - the machine is powering on....
2022-04-12 01:30:10 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:30:10 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:30:10 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:30:13 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-12 01:30:13 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:30:21 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11

2022-04-12 01:30:25 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.465 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.190 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.208 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.201 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.194 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 90ms
rtt min/avg/max/mdev = 0.190/0.251/0.465/0.108 ms

.2022-04-12 01:30:33 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:30:33 - transport.py - [line:1819] - paramiko.transport - ERROR - Socket exception: Connection reset by peer (104)
2022-04-12 01:30:33 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:30:33 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:30:33 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:30:33 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 01:30:36 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:30:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:30:44 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-12 01:30:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:30:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-12 01:30:53 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:31:01 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-12 01:31:01 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:31:06 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-12 01:31:06 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-12/test_PI_Networking_ConnectivityShutdown_01_18_22
2022-04-12 01:31:09 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-12 01:31:09 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:31:14 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-12 01:31:14 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-12 01:31:14 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-12 01:31:14 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_ConnectivityShutdown case ============================
                             [100%]

=================================== FAILURES ===================================
_____________ Test_Nics.test_gold_PI_Networking_SRIOV_CheckDevice ______________

self = <Tests.testNics.test_nics.Test_Nics object at 0x7f09028577b8>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicAll
    @pytest.mark.nicGold
    @pytest.mark.nicSilver
    @pytest.mark.nicSimple
    @pytest.mark.test
    def test_gold_PI_Networking_SRIOV_CheckDevice(self):
        cmd = "echo 4 > /sys/class/net/{}/device/sriov_numvfs".format(self.sut_conf["pf1"])
        stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, cmd)
        time.sleep(5)
        if stdout or stderr:
            assert 200 == 400
        sut2_ip = "192.168.10.11"
        self.cn_test_sut.ssh_exec_cmd(self.client2, "ifconfig {} {}\n".format(self.pkg_generator["pf1"], sut2_ip))
        port_up_cmd = "ifconfig {} up"
        manual_ip_cmd = "ifconfig {}{} 192.168.{}.{}"
        ping_ipv4_cmd = "ping -c 5 192.168.{}.{}"
        ping_ipv6_cmd = "ping6 -c 5 -I {}{} {}"
        for i in range(4):
            vf = "v"+str(i)
            ip_part = str(int(sut2_ip.split(".")[3])+i+1)
            time.sleep(5)
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, "ifconfig {}{} down".format(self.sut_conf["pf1"], vf))
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, "ifconfig {}{} up".format(self.sut_conf["pf1"], vf))
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, manual_ip_cmd.format(self.sut_conf["pf1"], vf,sut2_ip.split(".")[2] ,ip_part))
            if stdout or stderr:
                assert 200 == 400
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client2, ping_ipv4_cmd.format(sut2_ip.split(".")[2], ip_part))
            time.sleep(3)
            if stdout.find("received, 0% packet loss") == -1:
>               assert 200 == 400
E               assert 200 == 400

npxtester/Tests/testNics/test_nics.py:184: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-04-12 01:08:06 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-12/test_gold_PI_Networking_SRIOV_CheckDevice_01_08_06/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-04-12/test_gold_PI_Networking_SRIOV_CheckDevice_01_08_06
2022-04-12 01:08:06 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-12 01:08:06 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-12 01:08:06 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-12 01:08:06 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:08:06 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:08:06 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-12 01:08:06 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-12 01:08:06 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-12 01:08:06 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-12 01:08:06 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-12 01:08:09 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:08:09 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:08:18 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-12 01:08:18 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-12 01:08:23 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-12 01:08:23 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-12 01:08:26 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-12 01:08:26 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-12 01:08:31 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-12 01:08:31 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-12 01:08:34 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-12 01:08:46 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:08:51 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-12 01:08:51 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-12 01:08:54 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-12 01:08:58 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:09:03 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-12 01:09:06 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-12 01:09:06 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:09:14 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-12 01:09:14 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-12 01:09:22 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-12 01:09:22 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:09:27 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-12 01:09:30 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-12 01:09:31 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-12 01:09:39 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-12 01:09:39 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6461415522  bytes 9782567142382 (8.8 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 249370683  bytes 18516710671 (17.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


----------------------------- Captured stdout call -----------------------------
2022-04-12 01:09:47 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-12 01:09:47 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:00 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11

2022-04-12 01:10:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:13 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 down
2022-04-12 01:10:13 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:21 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 up
2022-04-12 01:10:21 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:29 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 192.168.10.12
2022-04-12 01:10:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:10:37 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.12
2022-04-12 01:10:41 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.458 ms
64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.196 ms
64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.189 ms
64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.191 ms
64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.223 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4081ms
rtt min/avg/max/mdev = 0.189/0.251/0.458/0.104 ms

2022-04-12 01:10:57 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 down
2022-04-12 01:10:58 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:11:06 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 up
2022-04-12 01:11:06 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:11:14 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 192.168.10.13
2022-04-12 01:11:14 - sshInstance.py - [line:54] - root - INFO - 
2022-04-12 01:11:22 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.13
2022-04-12 01:11:28 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
From 192.168.10.11 icmp_seq=1 Destination Host Unreachable
From 192.168.10.11 icmp_seq=2 Destination Host Unreachable
From 192.168.10.11 icmp_seq=3 Destination Host Unreachable
From 192.168.10.11 icmp_seq=4 Destination Host Unreachable
From 192.168.10.11 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 4116ms
pipe 4

------------------------------ Captured log call -------------------------------
INFO     root:sshInstance.py:50 echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens787f2 192.168.10.11

INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v0 down
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v0 up
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v0 192.168.10.12
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ping -c 5 192.168.10.12
INFO     root:sshInstance.py:54 PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.458 ms
64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.196 ms
64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.189 ms
64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.191 ms
64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.223 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4081ms
rtt min/avg/max/mdev = 0.189/0.251/0.458/0.104 ms

INFO     root:sshInstance.py:50 ifconfig ens785f2v1 down
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v1 up
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v1 192.168.10.13
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ping -c 5 192.168.10.13
INFO     root:sshInstance.py:54 PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
From 192.168.10.11 icmp_seq=1 Destination Host Unreachable
From 192.168.10.11 icmp_seq=2 Destination Host Unreachable
From 192.168.10.11 icmp_seq=3 Destination Host Unreachable
From 192.168.10.11 icmp_seq=4 Destination Host Unreachable
From 192.168.10.11 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 4116ms
pipe 4
=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:88
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_iperf_client_4hour.py:88: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:137
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_iperf_client_4hour.py:137: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:70
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:109
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:109: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:110
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:565
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:565: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:691
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:691: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:931
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:931: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_network_sriov_stress.py:1067
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:1067: PytestUnknownMarkWarning: Unknown pytest.mark.nicIperfTcpSUTGuestStress - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicIperfTcpSUTGuestStress

npxtester/Tests/testNics/test_network_sriov_stress.py:1068
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:1068: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:102
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:102: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:128
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:128: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:158
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:158: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:159
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:159: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:244
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:244: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:270
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:270: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:313
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:313: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_simple_report.html -
=========================== short test summary info ============================
FAILED npxtester/Tests/testNics/test_nics.py::Test_Nics::test_gold_PI_Networking_SRIOV_CheckDevice
===== 1 failed, 7 passed, 11 deselected, 18 warnings in 3924.77s (1:05:24) =====
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4Ix6O+hSs/J+WYhicXoVNFsq2LY7NwaGm8aSejP8W5cDAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRgg5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77aCbP8BPqPtor0AAAA=[0m[Pipeline] }
[8mha:////4GHchb786wl688erDJjASqsmJ9Hr4hPtHO8ytwYHX0MzAAAAoh+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6IkMhon14bFL6hQa6F5D9uHMPlF/pr/IJHEyTvdc5bzemOdIo4cnWotdZ5S7VUfBjc/NXLsroFH1fJF1UyJg1WVHStu7GnBisViWZZjpbGxVAdOnpxgq1vzMEUw5IqzxNkdNHLf3PFEpueymChfmCJ2t6FJTL/O3wCmvhfk5V6QlR9NTCl7vQAAAA==[0m[Pipeline] // node
[8mha:////4N3Zwv+i78+NyT5x5Iut/zlc/vu2cqXlCZqUPy9nGyYaAAAAoh+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6IkMhon14bFL6hQa6F5D9uHMPlF/pr/IJHEyTvdc5bzemOdIo4cnWotdZ5S7VUfBjc/NXLsroFH1fJF1UyJg1WVHStu7GnBisViWZZjpbGxVAdOnpxgq1vzMEUw5IqzxNkdNHLf3PFEpueymChfmCJ2t6FJTL/O3wCmvhfkZSnI9h9rVU4xvQAAAA==[0m[Pipeline] End of Pipeline
ERROR: script returned exit code 1
Finished: FAILURE
